doc/rados/operations/stretch-mode.rst

```diff
@@ -8,110 +8,114 @@ Stretch Clusters
+specific pools to be distributed across two or more data centers.
+This is done by executing the ``ceph osd pool stretch set`` command on each desired pool,
+contrasted with a cluster-wide strategy with *stretch mode*.
 See :ref:`setting_values_for_a_stretch_pool`
 
-Use ``stretch mode`` when you have exactly ``two data centers`` and require a uniform
-configuration across the entire cluster. Conversely, opt for a ``stretch pool``
-when you need a particular pool to be replicated across ``more than two data centers``,
-providing a more granular level of control and a larger cluster size.
+Use stretch mode when you have exactly two data centers and require a uniform
+configuration across the entire cluster. Conversely, opt for a stretch pool
+when you need only a particular pool to be replicated across more than two data centers,
+providing a more granular level of control.
 
 Limitations
 -----------
 
-Individual Stretch Pools do not support I/O operations during a netsplit
+Individual stretch pools do not support I/O operations during a netsplit
 scenario between two or more zones. While the cluster remains accessible for
-basic Ceph commands, I/O usage remains unavailable until the netsplit is
-resolved. This is different from ``stretch mode``, where the tiebreaker monitor
-can isolate one zone of the cluster and continue I/O operations in degraded
+basic Ceph commands, I/O remains unavailable until the netsplit is
+resolved. This is different from stretch mode, where the tiebreaker monitor
+can isolate one CRUSH ``datacenter`` and serve I/O operations in degraded
 mode during a netsplit. See :ref:`stretch_mode1`
 
-Ceph is designed to tolerate multiple host failures. However, if more than 25% of
-the OSDs in the cluster go down, Ceph may stop marking OSDs as out which will prevent rebalancing
-and some PGs might go inactive. This behavior is controlled by the ``mon_osd_min_in_ratio`` parameter.
-By default, mon_osd_min_in_ratio is set to 0.75, meaning that at least 75% of the OSDs
-in the cluster must remain ``active`` before any additional OSDs can be marked out.
-This setting prevents too many OSDs from being marked out as this might lead to significant
-data movement. The data movement can cause high client I/O impact and long recovery times when
-the OSDs are returned to service. If Ceph stops marking OSDs as out, some PGs may fail to
+Ceph is designed to tolerate multiple component failures. However, if more than 25% of
+the OSDs in the cluster go down, Ceph may stop marking OSDs ``out``, which prevents rebalancing
+and may result in PGs becoming ``inactive``. This behavior
+is controlled by the ``mon_osd_min_in_ratio`` option.
+The default value is ``0.75``, meaning that at least 75% of the OSDs
+in the cluster must be ``active`` for any additional OSDs to be marked out.
+This setting prevents too many OSDs from being marked out as this might lead to
+cascading failures and an impactful thundering herd of data movement. This can
```

Comment:
I don't think I can permit the reference to Marshall to stand here.

Reply:
Nothing against Marshall.

Reply:
? Or Will & Holly?